code/modules/food/nuggets.dm

```diff
@@ -0,0 +1,16 @@
+/obj/item/food/nugget
+	name           = "chicken nugget"
+	icon           = 'icons/obj/food/nuggets.dmi'
+	icon_state     = ICON_STATE_WORLD
+	nutriment_desc = "mild battered chicken"
+	nutriment_amt  = 6
+	nutriment_type = /decl/material/solid/organic/meat/chicken
+	material       = /decl/material/solid/organic/meat/chicken
+	bitesize       = 3
+
+/obj/item/food/nugget/Initialize()
+	. = ..()
+	var/shape = pick("lump", "star", "lizard", "corgi")
+	desc = "A chicken nugget vaguely shaped like a [shape]."
+	icon_state = "[icon_state]-[shape]"
```

Comment:
i feel like this should probably be in an on_update_icon override and use the world/inventory state?

Reply:
if an inventory state is ever added to the file it'd break without any corresponding code changes. not a big deal though